map/src/test/java/openlr/map/teststubs/OpenLRMapDatabaseAdaptor.java

```diff
@@ -0,0 +1,154 @@
+package openlr.map.teststubs;
+
+import com.vividsolutions.jts.geom.Envelope;
```

Comment:
same here for the vividsolutions... maybe this dependency comes from other dependency.